packages/luit.rb

```diff
@@ -0,0 +1,40 @@
+# Adapted from Arch Linux luit PKGBUILD at:
+# https://github.com/archlinux/svntogit-packages/raw/packages/luit/trunk/PKGBUILD
+
+require 'package'
+
+class Luit < Package
+  description 'Filter that can be run between an arbitrary application and a UTF-8 terminal emulator'
+  homepage 'https://invisible-island.net/luit/luit.html'
+  version '2_0_20230201'
+  license 'custom'
+  compatibility 'all'
+  source_url 'https://github.com/ThomasDickey/luit-snapshots.git'
+  git_hashtag 'v2_0_20230201'
```

Comment:
Use `git_hashtag "v#{version}"` or `git_hashtag 'v' + version`?

Reply:
The tag name varies the letters used... Otherwise, yes definitely.

Reply:
How does it vary?  Not understanding.

Reply:
![Screenshot_20230207-200213.png](https://user-images.githubusercontent.com/1096701/217402038-530c87e8-f1f5-42cd-a107-811d40762b2b.png)

Reply:
Not sure what the `t` means but the `v` versions look consistent to me.

Reply:
Will we ever use a `t` release?

Reply:
Maybe not.  Changed the file as requested.